src/System.Buffers/src/System/Buffers/ArrayPoolEventSource.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace System.Buffers
+{
+    public sealed class ArrayPoolEventSource : EventSource
+    {
+        public ArrayPoolEventSource() : base() { }
+
+        [Event(1, Message = "Rented Buffer {0} of size {1} from Bucket {2}", Level = EventLevel.Informational)]
```

Comment:
It should be possible.  EventSource has its own design for localization, e.g. https://github.com/dotnet/coreclr/blob/bc146608854d1db9cdbcc0b08029a87754e12b49/src/mscorlib/src/System/Threading/Tasks/TPLETWProvider.cs#L27